R/pkg/R/functions.R

```diff
@@ -259,6 +259,20 @@ setMethod("column",
           function(x) {
             col(x)
           })
+#' corr
+#'
+#' Computes the Pearson Correlation Coefficient for two Columns.
+#'
+#' @rdname corr
+#' @name corr
+#' @family math_funcs
+#' @export
+#' @examples \dontrun{corr(df$c, df$d)}
+setMethod("corr", signature(x = "Column", col1 = "Column", col2 = "missing", method = "missing"),
```

Comment:
right, I like the approach of changing the existing generic definition.

perhaps we should align the method signature with the stats::cor
```
cor(x, y = NULL, use = "everything",
    method = c("pearson", "kendall", "spearman"))
```

do you know why we decide to name it `corr` (vs. `cor`) in other places?